integration_tests/cairozero_test.go

```diff
@@ -74,11 +74,9 @@ func TestCairoZeroFiles(t *testing.T) {
 			continue
 		}
 
-		var path string
+		path := filepath.Join(root2, dirEntry.Name())
 		if _, found := testFiles1Map[dirEntry.Name()]; found {
 			path = filepath.Join(root1, dirEntry.Name())
-		} else {
```

Comment:
why do you remove this?
its part of integration tests, it files without cairo zero hints

Reply:
Again, its cleaner like this. 

This is the initializer : `path := filepath.Join(root2, dirEntry.Name())`
And this updates it if it is in root1 : 		
```
if _, found := testFiles1Map[dirEntry.Name()]; found {
    path = filepath.Join(root1, dirEntry.Name())
}
```

Reply:
i dont get it, why would you do with no conditiion

```
path := filepath.Join(root2, dirEntry.Name())
```
?

Reply:
Because it gets updated if its root1 and if its not, it remains root2.